app/components/find/courses/training_locations/view.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Find
+  module Courses
+    module TrainingLocations
+      class View < ViewComponent::Base
+        include PublishHelper
+        include PreviewHelper
+
+        attr_reader :course, :preview
+
+        def initialize(course:, preview: false)
+          @course = course
+          @preview = preview
+          super
+        end
+
+        def placements_url
+          if preview
+            placements_publish_provider_recruitment_cycle_course_path(
+              course.provider_code,
+              course.recruitment_cycle_year,
+              course.course_code
+            )
+          else
+            find_placements_path(course.provider_code, course.course_code)
+          end
+        end
+
+        def potential_placements_text
+          "#{course.sites.size} potential placement #{course.sites.many? ? 'locations' : 'location'}"
```

Comment:
CC @RebeccaCryan (for when you return).

We renamed all user facing content on Publish from "Locations" to "Schools", should we be consistent and use "Schools" here too?

<img width="749" alt="image" src="https://github.com/user-attachments/assets/0af84c0a-ee0c-455f-8007-2288df5adb6d">

Reply:
[Rails translations](https://guides.rubyonrails.org/i18n.html#pluralization) can handle all of the "none", "one" and "many" scenarios. Are there benefits to implementing the logic outside of translations?

Reply:
Thanks, didn't mean to reinvent the wheel here 😅  

I've gone with @inulty-dfe suggestion of using `pluralize` rails method.

https://github.com/DFE-Digital/publish-teacher-training/pull/4522/commits/47bac84a3f64eceef9f64617eb993b68ae307815